NoSQL/ce/start-kvlite.sh

```diff
@@ -1,7 +1,12 @@
+#!/bin/bash
+#
 # Copyright (c) 2022 Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
-#
 
-KVHOST=`hostname`
-java -jar lib/kvstore.jar kvlite  -secure-config disable -root /kvroot &
-java -jar lib/httpproxy.jar -helperHosts $KVHOST:5000 -storeName kvstore -httpPort $KV_PROXY_PORT -verbose true
++set -ex
+
+KVHOST=$HOSTNAME
+_KV_HARANGE=`echo $KV_HARANGE | sed 's/\-/\,/g' `
+_KV_SERVICERANGE=`echo $KV_SERVICERANGE | sed 's/\-/\,/g' `
+java -jar lib/kvstore.jar kvlite -secure-config disable -root /kvroot -host $KVHOST -port $KV_PORT -admin-web-port $KV_ADMIN_PORT -harange $_KV_HARANGE -servicerange $_KV_SERVICERANGE &
+java -jar lib/httpproxy.jar -helperHosts $KVHOST:$KV_PORT -storeName kvstore -httpPort $KV_PROXY_PORT -verbose true
```

Comment:
Also, I had to insert a loop between the two commands so that the proxy didn't exit and shutdown the container prematurely. I went with this:

```bash
while [ ! -d /kvroot/kvstore/log ]; do
    echo "Waiting for kvstore to start..."
    sleep 1
done
```

Which seems to be enough. If there's a better way to get the proxy process not to give up so quickly, use that. :)

Reply:
Thanks for catching this. I will add this to a next version

```
while java -jar lib/kvstore.jar ping -host $HOSTNAME -port 5000  >/dev/null 2>&1 ; [ $? -ne 0 ];do
    echo "Waiting for kvstore to start..."
    sleep 1
done
```